File: code_samples/python/v1.0.0/get_payment_status.py
from tatrapayplus import TatrapayPlusClient

client = TatrapayPlusClient(
    "your-client-id",
    "your-client-secret",
)
payment_id = "b54afd37-5bb9-4080-9416-5ec450779087"  # Retrieved from create payment intent
payment_status = client.get_payment_status(payment_id)

File: code_samples/php/v1.0.0/get_payment_status.php
<?php
use Tatrapayplus\TatrapayplusApiClient\Api\TatraPayPlusAPIApi;

$tatrapayplus_api = new TatraPayPlusAPIApi(
    "your-client-id",
    "your-client-secret",
);
$payment_id = 'b54afd37-5bb9-4080-9416-5ec450779087'; // Retrieved from initiatePayment

$response = $tatrapayplus_api->getPaymentIntentStatus($payment_id);

$response_obj = $response['object'];
$simple_status = $response_obj->getSimpleStatus();
$status = $response_obj->getStatus();
$payment_method = $response_obj->getSelectedPaymentMethod();
$authorization_status = $response_obj->getAuthorizationStatus();